NDTensors/src/blocksparse/blocksparsetensor.jl

```diff
@@ -49,7 +49,7 @@ Construct a block sparse tensor with uninitialized memory
 from indices and locations of non-zero blocks.
 """
 function BlockSparseTensor(::UndefInitializer, blockoffsets, inds)
-  return BlockSparseTensor(Float64, undef, blockoffsets, inds)
+  return BlockSparseTensor(NDTensors.default_eltype(), undef, blockoffsets, inds)
```

Comment:
```suggestion
  return BlockSparseTensor(default_eltype(), undef, blockoffsets, inds)
```
and same below. If we are in the `NDTensors` library there shouldn't be a need to qualify a function with `NDTensors.`, unless I'm missing something here.

Reply:
Makes sense, I removed all instances in this file. Thanks!